src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -229,7 +234,7 @@ void KeyDropDownControl::SetSelectionHandler(Grid& table, StackPanel shortcutCon
             }
 
             // Reset the buffer based on the new selected drop down items. Use static key code list since the KeyDropDownControl object might be deleted
-            std::vector selectedKeyCodes = KeyboardManagerHelper::GetKeyCodesFromSelectedIndices(GetSelectedIndicesFromStackPanel(parent), GetKeyCodeList(true, colIndex == 1));
+            std::vector<int32_t> selectedKeyCodes = GetSelectedCodesFromStackPanel(parent);
```

Comment:
We also need to do the count_if check here https://github.com/mykhailopylyp/PowerToys/blob/a5f0b5c58c0ce59204f884f175df607110466723/src/modules/keyboardmanager/ui/KeyDropDownControl.cpp#L245 and https://github.com/mykhailopylyp/PowerToys/blob/a5f0b5c58c0ce59204f884f175df607110466723/src/modules/keyboardmanager/ui/KeyDropDownControl.cpp#L344